qr_generator/lib/utils/api_client.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:dio/dio.dart';
+
+class ApiClient {
+  late Dio dio;
+  ApiClient() {
+    setupDio();
+  }
+
+  setupDio() async {
```

Comment:
**fix**: prefer doing this in a factory constructor. No need for `async`

Reply:
Done!